index.html

```diff
@@ -1412,6 +1412,104 @@ <h3>Biometrics</h3>
+      new DID method, or implementing a solution that incorporates DIDs.
+    </p>
+
+    <p>
+      The <a 
+      href="https://www.w3.org/2001/tag/doc/ethical-web-principles/">W3C TAG
+      Ethical Web Principles</a> document is a reliable guide for making assessments
+      and weighing ethical considerations of various technologies.
+    </p>
+
+    <section class="informative">
+      <h3>Broad Ethical Principles</h3>
+      <p>
+        Decentralized Identifiers may come to underpin much of our digital life. This 
+        may include public social and career personas, as well as more private personas 
+        shared among friends and family. Identifiers representing these personas and 
+        relationships may be some of the most important in our lives, so great care 
+        will be taken when choosing an identifier system to support these critical 
+        activities. As with all things, the appropriate and ethical use of technology
+        will be strongly considered  when implementing items related to DIDs.
+      </p>
+
+      <p>
+        As noted in the <a
+        href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#principles">
+        Principles</a> section of the Ethical Web Principles, there are key goals that
+        apply to all Web standards and technologies. DIDs are explicitly intended to
+        help users get closer to several of these goals, especially the following:
+        <ul>
+          <li>
+            <a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#privacy">
+              Privacy
+            </a>
+          </li>
+          <li>
+            <a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#control">
+              Individual Control
+            </a>
+          </li>
+          <li>
+            <a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#multi">
+              Device Independence
+            </a>
+          </li>
+        </ul>
+      </p>
+    </section>
+
+    <section class="informative">
+      <h3>Environmental Considerations</h3>
+      <p class="advisement">
+        The following section reflects the views of some members of the working group.
+        Additional PRs are welcome from the working group with additional points of view.
+      </p>
+
+      <section class="informative">
+        <h3>Environmental Principles as Applied to DID method development</h3>
+
+        <p>
+          The guiding principle that 
+          <a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#sustainable">the
+          web must be an environmentally sustainable platform</a> should be followed.
+          <br />
+          When implementing or utilizing a DID method, consideration should be given
+          to the environmental impacts of any underlying technologies. 
+        </p>
+
+        <p>
+          Utilizing or authoring DID methods that compromise on
```

Comment:
This claim presents a false dichotomy of a tradeoff between environmental concerns and security. There is no such tradeoff inherent in DIDs. This is not zooko's triangle.

Reply:
Section has been removed - @jandrieu are you ok with me resolving this and the other "outdated" feedback now that that tradeoff section is pulled?